table.go

```diff
@@ -562,6 +562,11 @@ func (t *Table) SetContent(content TableContent) *Table {
 	return t
 }
 
+// GetContent returns current table's content
```

Comment:
https://github.com/rivo/tview/blob/master/CONTRIBUTING.md:

> Start all sentences upper-case [...] and **end them with a period**.

Also, I think you mean *"the table's current content."*